smurfs/src/components/SmurfsList.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import {connect} from 'react-redux';
+import {getSmurfs} from '../actions/getSmurfs';
+import Smurf from './Smurf';
+
+
+const mapStateToProps = state => ({
+    smurfs: state.smurfs
+});
```

Comment:
Seeing mstp in a place reminded me of something, if your curious of how to structure your component files I'd recommend looking into the _below the fold_ technique.